sdk/typescript/introspector/scanner/abtractions/property.ts

```diff
@@ -5,7 +5,7 @@ import { UnknownDaggerError } from "../../../common/errors/UnknownDaggerError.js
 import { FieldTypeDef, TypeDef } from "../typeDefs.js"
 import { typeToTypedef } from "../utils.js"
 
-const PROPERTY_DECORATOR = "field"
+const PROPERTY_DECORATOR = "func"
```

Comment:
This isn't enough because now `@field` doesn't work.

Reply:
Oh yeah oups, thanks for catching that!